templates/cert.cnf.erb

```diff
@@ -14,7 +14,7 @@ default_md              = sha256
 default_keyfile         = privkey.pem
 distinguished_name      = req_distinguished_name
 prompt                  = no
-<% if @altnames and not @altnames.empty? -%>
+<% if (@altnames and not @altnames.empty?) || (@extkeyusage and not @extkeyusage.empty?) -%>
```

Comment:
It might be easier to create a variable for that (e.g. `Boolean use_req`) in the Puppet code.

Reply:
Done.